distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/cli/InstallPluginActionTests.java

```diff
@@ -581,7 +581,8 @@ public void testJarHell() throws Exception {
         assumeTrue("real filesystem", isReal);
         Path pluginDirectory = createPluginDir(temp);
         writeJar(pluginDirectory.resolve("other.jar"), "FakePlugin");
-        PluginDescriptor pluginZip = createPluginZip("fake", pluginDirectory); // adds plugin.jar with FakePlugin
+        InstallablePlugin pluginZip = createPluginZip("fake", pluginDirectory); // adds plugin.jar with
```

Comment:
not even a nit: Did Spotless reformat this line-ending comment? I don't think I've seen a comment at the end of a line spill over to the next line in our code base. (This was the only surprise in the diff, but it's fine with me if it stays.)

Reply:
I think what happened was in an initial commit for this, I had not renamed this PluginDescriptor, and so it had a fully qualified name, which pushed the line length out, and did this odd comment splitting. Then when I renamed this to InstallablePlugin, it was no longer too long. I've fixed it now and spotless seems happy.